lib/granite/form/types/boolean.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Granite
+  module Form
+    module Types
+      class Boolean < Object
+        MAPPING = {
+          1 => true,
+          0 => false,
+          '1' => true,
+          '0' => false,
+          't' => true,
+          'f' => false,
+          'T' => true,
+          'F' => false,
+          true => true,
+          false => false,
+          'true' => true,
+          'false' => false,
+          'TRUE' => true,
+          'FALSE' => false,
+          'y' => true,
+          'n' => false,
+          'yes' => true,
+          'no' => false
```

Comment:
Y and N or YES and NO?

Reply:
I don't think why upcase are not parsed, I just copy pasted the constant